spacemacs/compile-comments.vim

```diff
@@ -7,20 +7,20 @@ nnoremap <leader>cc    :action CompileDirty<CR>
 vnoremap <leader>cc    :action CompileDirty<CR>
 
 " Comment lines
-nnoremap <leader>cl    :action CommentByLineComment<CR>:action VimVisualExitMode<CR>
-vnoremap <leader>cl    :action CommentByLineComment<CR>:action VimVisualExitMode<CR>
+nnoremap <leader>cl    :action CommentByLineComment<CR>
```

Comment:
This is better, because after `CommentByLineComment` you aren't in visual mode and so `VimVisualExitMode` is not necessary